src/components/NewLocationPage/CountyMap/CountyMap.tsx

```diff
@@ -73,6 +74,12 @@ const CountyMap: React.FC<{ region: Region }> = React.memo(({ region }) => {
     }
   };
 
+  const { pathname } = useLocation();
+
+  useEffect(() => {
```

Comment:
I think a small comment like "resets maptype to vaccinations on page navigation" would be helpful.  may be obvious but took me a sec